lessons/ro/chapter_9.yaml

```diff
@@ -126,17 +126,17 @@
     În mod implicit, membrii unei cutii nu sunt accesibili din afara cutiei. Noi
     facem membrii unei
 
-    cutii accesibilă prin marcarea lor ca `pub` în modulul rădăcină (în engleză *root module*) al
+    cutii accesibili prin marcarea lor ca `pub` în modulul rădăcină (în engleză *root module*) al
     cutiei dvs. (`lib.rs` sau `main.rs`).
 - title: Vizibilitatea Structurilor
   content_markdown: >
     La fel ca și funcțiile, structurile pot declara ceea ce doresc să fie expus
     în afara modulului lor folosind `pub`.
   code: >-
-    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=%2F%2F%20SeaCreature%20struct%20will%20be%20usable%20outside%20of%20our%20module%0Apub%20struct%20SeaCreature%20%7B%0A%20%20%20%20pub%20animal_type%3A%20String%2C%0A%20%20%20%20pub%20name%3A%20String%2C%0A%20%20%20%20pub%20arms%3A%20i32%2C%0A%20%20%20%20pub%20legs%3A%20i32%2C%0A%20%20%20%20%2F%2F%20let's%20keep%20our%20weapon%20private%0A%20%20%20%20weapon%3A%20String%2C%0A%7D%0A
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2021
```

Comment:
Looks like you deleted the part of the URL which contains the actual code to be displayed